test/test_files/randomprime_expected_data_one_way_door.json

```diff
@@ -34,24 +34,8 @@
         "forceFusion": false
     },
     "gameConfig": {
-        "resultsString": "4.4.0.dev211 | Seed Hash - Geemer Shriekbat Fission (AAAAAAAA)",
-        "bossSizes": {
-            "parasiteQueen": 0.44977572662984067,
-            "incineratorDrone": 0.5455729671029474,
-            "adultSheegoth": 0.7089159456686496,
-            "thardus": 1.6623366020623678,
-            "elitePirate1": 2.2878011900309114,
-            "elitePirate2": 0.30400815014892685,
-            "elitePirate3": 0.9231433057954543,
-            "phazonElite": 0.6409823670291537,
-            "omegaPirate": 0.36457831494081505,
-            "Ridley": 0.7414786873418531,
-            "exo": 0.22667724053748933,
-            "essence": 0.14812015272534665,
-            "flaahgra": 2.0275229604319627,
-            "platedBeetle": 0.42416872030226377,
-            "cloakedDrone": 0.6282245815635213
-        },
+        "resultsString": "4.6.0.dev146 | Seed Hash - Geemer Shriekbat Fission (AAAAAAAA)",
+        "bossSizes": null,
```

Comment:
The preset corresponding to this `json` output specifies:
```json
"random_boss_sizes": true,
```

It does NOT specify random attributes.

The expected behavior is that a random boss size dict is placed here.